Cause models for "residual maternal disorders" (#1789)
ihmeuw/vivarium_research: docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/partial_term_causes.rst

partial term pregnancy outcomes according to the incidence rates of the GBD causes of
ectopic pregnancy and abortion/miscarriage, we assume that 100% of partial term pregnancies
in the MNCNH simulation are "incident cases" and therefore all partial term pregnancies
are subject to the case fatality rate and YLD rate associated with the partial term 
pregnancy maternal disorders cause as defined below.

Assumptions and Limitations
+++++++++++++++++++++++++++

- We do not model mortality or morbidity due to ectopic pregnancies or abortion/miscarriage separately and rather model a summary measure across both of these causes. Therefore, if we were ever to model an intervention that acts on either of these subcauses, we would need to update our modeling strategy to separate these subcauses from this single cause model.

Cause Model Diagram
+++++++++++++++++++

.. graphviz::

    digraph hemorrhage_decisions {
        rankdir = LR;
        ptp [label="partial term\npregnancy, post\nantenatal models", style=dashed]
        alive [label="parent did not die\nof partial term pregnancy\nmaternal disorders"]
        dead [label="parent died of partial\nterm pregnancy maternal disorders"]

>         PTPMD [label="affected with\mpartial term pregnancy\nmaternal disorders"]

Reviewer: ```suggestion
        PTPMD [label="affected with\npartial term pregnancy\nmaternal disorders"]
```

    * - cfr
      - case fatality rate
      - The probability that a simulant with partial term pregnancy maternal disorders dies of that partial term pregnancy maternal disorders

Data Tables
+++++++++++

.. note::

    All quantities pulled from GBD in the following table are for a
    specific year, sex, age group, and location unless otherwise noted
    (e.g., SBR). Our simulation only includes pregnant women of
    reproductive age, so the sex will always be female. However, even
    though all of our simulants will be pregnant, we still pull each
    quantity for *all* females in a given year, age group, and location,
    because this is the default behavior of GBD. Since we are using the
    same total population in all the denominators, the person-time will
    cancel out in the above calculations to give us the probabilities we
    want.

.. list-table:: Data values and sources
    :header-rows: 1

    * - Variable
      - Definition
      - Value or source
      - Note
    * - ir
      - partial term pregnancy maternal disorders incidence risk per partial term pregnancy
      - 1

>       - Artifact of the modeling strategy that assings partial term pregnancy outcomes (in the :ref:`pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`) according to the incidence of the GBD causes included in this cause model document

Reviewer: ```suggestion
      - Artifact of the modeling strategy that assigns partial term pregnancy outcomes (in the :ref:`pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`) according to the incidence of the GBD causes included in this cause model document
```

        parameter to susceptible-population incidence rate using
        condition prevalence. Total population person-time is used in
        the denominator in order to cancel out with the person-time in
        the cause-specific mortality rate denominator.
    * - csmr_c374, csmr_c995
      - cause-specific mortality rate of ectopic pregnancy, abortion/miscarriage
      - deaths_c374 / population, deaths_c995 / population
      - Note that deaths / (average population for year) = deaths / person-time
    * - deaths_c374, deaths_c995
      - count of deaths due to ectopic pregnancy, abortion/miscarriage
      - codcorrect
      -
    * - population
      - average population in a given year
      - get_population
      - Specific to age/sex/location/year demographic group. Numerically
        equal to person-time for the year.
    * - yld_rate_c374, yld_rate_995
      - rate of ectopic pregnancy, abortion/miscarriage YLDs per person-year
      - como
      -
    * - ylds_per_case
      - YLDs per case of partial term pregnancy maternal disorders
      - (yld_rate_c374 + yld_rate_c995) / (incidence_c374 + incidence_c995)
      - 

> - The ylds_per_case parameter should be applied to all simulants affected by residual maternal disorders (all full term pregnancies)

Reviewer: ```suggestion
- The ylds_per_case parameter should be applied to all simulants affected by partial term maternal disorders (all partial term pregnancies)
```
